src/pages/home/home.html

```diff
@@ -1,41 +1,87 @@
-
 <div class="toolbar-class">
   <div>
     <ion-header>
       <ion-navbar color = "dark">
-        <img src="../../assets/imgs/docent-logo-f-n-l-1.jpg">
-        <button ion-button color="dark" right>
-            Assesment
-        </button>
-        <button ion-button color="dark">
-            Help
-        </button>
+        <div class = "container-toolbar">
+          <img src="../../assets/imgs/docent-logo-f-n-l-1.jpg">
+          <div class = "container-tbbuttons">
+            <button ion-button color="dark">
+                Assesment
+            </button>
+            <button ion-button color="dark">
+                Help
+            </button>
+          </div>
+        </div>
       </ion-navbar>
     </ion-header>
   </div>
 </div>
-<ion-content>
-  <p>Test</p>
-  <p>Test</p>
+<ion-content padding>
+  <br>
+  <br>
   <h1>Start New Assessment</h1>
   <font color="grey">Use this option to start a new assessment</font>
   <br><br>
   <h6>Team Members</h6>
   <form>
-    <input placeholder="Member Name">
-    <input placeholder="Member Role">
-    <ion-item>
-    <ion-label>Member Name</ion-label>
-    <ion-input type="text" name="title"></ion-input>
-    </ion-item>
-    <ion-item>
-      <ion-label>Member Role</ion-label>
-      <ion-textarea  name="description"></ion-textarea>
+    <div class="members">
+      <ion-item>
+        <ion-input type="text" name="memName" placeholder = "Member Name"></ion-input>
+        <ion-input type="text" name="memRole" placeholder = "Member Role"></ion-input>
       </ion-item>
+      <div class = "buttons">
+        <button (click)="removeMember()">
```

Comment:
Good start. By demo time, these functions should be working and actually updating the display.